src/config/dynamic-theme-fixes.config

```diff
@@ -3584,6 +3584,65 @@ IGNORE IMAGE ANALYSIS
 .is2d .queen.black
 .is2d .rook.black
 
+================================
+
+lightning.force.com
```

Comment:
Invalid URL

Reply:
It's intranet page you won't be able to access it unfortunately.